index.js

```diff
@@ -88,6 +88,8 @@ function debounce(function_, wait = 100, options = {}) {
 		debounced.clear();
 	};
 
+	debounced.isDebouncing = () => Boolean(timeoutId);
```

Comment:
Would be better to do an explicit check
```suggestion
	debounced.isDebouncing = () => timeoutId !== undefined;
```

Reply:
I just did it similarly to how it's done in index.js:57.
`if (!timeoutId) {
    timeoutId = setTimeout(later, wait);
}`
but it is really better to do as you indicated. thanks! I fixed it!